Tests.App/Services/UriHandlerTest.cs

```diff
@@ -0,0 +1,184 @@
+using Coder.Desktop.App.Models;
+using Coder.Desktop.App.Services;
+using Coder.Desktop.Vpn.Proto;
+using Google.Protobuf;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Serilog;
+
+namespace Coder.Desktop.Tests.App.Services;
+
+[TestFixture]
+public class UriHandlerTest
+{
+    [SetUp]
+    public void SetupMocksAndUriHandler()
+    {
+        Serilog.Log.Logger = new LoggerConfiguration().MinimumLevel.Debug().WriteTo.NUnitOutput().CreateLogger();
+        var builder = Host.CreateApplicationBuilder();
+        builder.Services.AddSerilog();
+        var logger = (ILogger<UriHandler>)builder.Build().Services.GetService(typeof(ILogger<UriHandler>))!;
```

Comment:
Creating an application builder for this seems excessive as it's only being used to create `ILogger` instances. Is there no way to just get an `ILogger` from serilog and pass that through without using application hosting?

Reply:
If there is, I haven't figured it out.